.github/workflows/ci.yml

```
@@ -10,13 +10,17 @@ on:
 
 jobs:
   ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ['18.x', '20.x', 'lts/*']
```

18.x is EOL..

---

Still one more week, but we can drop it.  https://nodejs.org/en/blog/release/v18.20.8
```suggestion
        node-version: [ '20.x', 'lts/*']
```
Should we add `latest` or just focus on LTS versions?
* https://github.com/actions/setup-node?tab=readme-ov-file#supported-version-syntax